test/test_discover.py

```diff
@@ -0,0 +1,87 @@
+"""Test for discover module
+
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+from typing import List, Dict
+
+import spdata.discover as disc
```

Comment:
Please import by full name, as in `from spdata import discover`. While this is fine for common abbreviations like `numpy → np`, there is some unnecessary mental overhead we could omit here.